pagination/templates/includes/_pagination.html

```diff
@@ -0,0 +1,15 @@
+{% load querystrings %}
+
+{% if paginator.page_range|length > 1 %}
+    <div class="pagination {{ border_class|default:"bordered" }}">
+        {% if page_obj.has_previous %}
+            <a class="prev" href="?{% querystring_update request page=page_obj.previous_page_number %}">Previous</a>
```

Comment:
I think it would be better to move the `?` into `querystring_update` until we do need that extra flexibility. It looks like a foot-gun currently.